onchain/rollups/contracts/consensus/quorum/Quorum.sol

```diff
@@ -3,32 +3,44 @@
 
 pragma solidity ^0.8.8;
 
-import {AccessControlEnumerable} from "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
-import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 import {PaymentSplitter} from "@openzeppelin/contracts/finance/PaymentSplitter.sol";
 
 import {AbstractConsensus} from "../AbstractConsensus.sol";
+import {IConsensus} from "../IConsensus.sol";
 import {IHistory} from "../../history/IHistory.sol";
 
 /// @title Quorum consensus
 /// @notice A consensus model controlled by a small set of addresses, the validators.
-/// @dev This contract uses several OpenZeppelin contracts:
-/// `AccessControlEnumerable`, `EnumerableSet`, and `PaymentSplitter`.
-/// For more information on those, please consult OpenZeppelin's official documentation.
-contract Quorum is AbstractConsensus, AccessControlEnumerable, PaymentSplitter {
-    using EnumerableSet for EnumerableSet.AddressSet;
-
-    /// @notice The validator role.
-    /// @dev Only validators can submit claims.
-    bytes32 public constant VALIDATOR_ROLE = keccak256("VALIDATOR_ROLE");
-
+///         In this version, the validator set is immutable.
+/// @dev This contract uses OpenZeppelin `PaymentSplitter`.
+///      For more information on `PaymentSplitter`, please consult OpenZeppelin's official documentation.
+contract Quorum is AbstractConsensus, PaymentSplitter {
     /// @notice The history contract.
     /// @dev See the `getHistory` function.
     IHistory internal immutable history;
 
-    /// @notice For each claim, the set of validators that agree
-    /// that it should be submitted to the history contract.
-    mapping(bytes => EnumerableSet.AddressSet) internal yeas;
+    // Quorum members
+    // Map an address to true if it's a validator
+    mapping(address => bool) public validators;
+    uint256 public immutable quorumSize;
+
+    // Quorum votes
```

Comment:
Since we had agreed to use the term `validator` rather than `member`, I will not rename the struct to `MemberSet` here.
Also I think it should be more about the votes rather than the validators. The set of votes changes for different claims but the set of validators doesn't change.

Reply:
> The set of votes changes for different claims but the set of validators doesn't change.

Yes, you're correct. Maybe what I had in mind was a subset of validators.

Reply:
> Also I think it should be more about the votes rather than the validators.

Fair enough! Still, don't you think it's quite weird to have a struct named after an adjective?
Maybe rename `Voted` to `Votes`?

Reply:
Agree. Will do